utils.go

```diff
@@ -111,7 +111,8 @@ func getAllFiles(paths []string, languages *DefinedLanguages, opts *ClocOptions)
 		vcsInRoot := isVCSDir(root)
 		err = filepath.Walk(root, func(path string, info os.FileInfo, err error) error {
 			if err != nil {
-				return err
+				fmt.Printf("%s\n", err)
```

Comment:
I want to output a message to stderr because it is an error message.
How about the following code?

```suggestion
				fmt.Fprintf(os.Stderr, "%s\n", err)
```

Reply:
Thanks for the suggestions. I've requested another review.